From e767a88e8d3316dd8f1a0087ed1a1d783f5c436a Mon Sep 17 00:00:00 2001
From: Tobias Mueller <muelli@cryptobitch.de>
Date: Tue, 27 Aug 2019 23:48:59 +0100
Subject: [PATCH 3/4] appdata: rename updatecontact to update_contact to be
 more compliant

The validator complained about the tag name and recommended the other.

Signed-off-by: Tobias Mueller <muelli@cryptobitch.de>
---
 wireshark.appdata.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wireshark.appdata.xml b/wireshark.appdata.xml
index d888e050a4..ed14f9ccc8 100644
--- a/wireshark.appdata.xml
+++ b/wireshark.appdata.xml
@@ -46,7 +46,7 @@
 	<screenshot type="default">https://www.wireshark.org/~morriss/Screenshot-Wireshark2.0-Http-cap.png</screenshot>
 	<screenshot>https://www.wireshark.org/~morriss/Screenshot-Wireshark2.0-Stevens-graph.png</screenshot>
     </screenshots>
-    <updatecontact>wireshark-dev_at_wireshark.org</updatecontact>
+    <update_contact>wireshark-dev_at_wireshark.org</update_contact>
     <releases>
         <release version="3.0.3" date="2019-07-17"/>
         <release version="3.0.2" date="2019-05-21"/>
-- 
2.17.1

